__tests__/helpers/BasicPointsAndBadges/ConnectExternalAccountActivity.test.ts

```diff
@@ -0,0 +1,115 @@
+        status: "unauthorized"
+    },
+    {
+        "id" : 4,
+        "provider": { "id": 44, "name": "Health Plan 1", "category": "Health Plan"} ,
+        status: "fetchComplete"
+    }];
+
+    return {
+        __esModule: true,
+        default: {
+            token: { access_token: '1' },
+            getCurrentLanguage: () => jest.fn(),
+            getExternalAccounts: jest.fn(() => {
+                return Promise.resolve(externalAccounts);
+            }),
+            on: jest.fn(),
+            off: jest.fn()
+        }
+    }
+});
+
+describe("ConnectExternalAccountActivity Awards", () => {
+    it("should award points for connecting a single external account", async () => {
+         const activity = {
+             type: "connectExternalAccount",
+             points: 10,
+             providerCategories: ["Health Plan"]
+         } as ConnectExternalAccountActivity;
+         const activityState = {
+             pointsAwarded: 0,
+             providersConnected: []
+         };
+         const newActivityState = await awardConnectExternalAccountActivityPoints(activity, activityState);
+         expect(newActivityState.pointsAwarded).toBe(10);
+         expect(newActivityState.providersConnected).toEqual([44]);
+    });
+
+    it("should award points for connecting multiple external accounts", async () => {
+        const activity = {
+            type: "connectExternalAccount",
+            points: 10,
+            providerCategories: ["Health Plan", 'Provider', 'Device Manufacturer']
+        } as ConnectExternalAccountActivity;
+        const activityState = {
+            pointsAwarded: 0,
+            providersConnected: []
+        };
+        const newActivityState = await awardConnectExternalAccountActivityPoints(activity, activityState);
+        expect(newActivityState.pointsAwarded).toBe(30);
+        expect(newActivityState.providersConnected).toEqual([11, 22, 44]);
+   });
+
+    it("should NOT award points for multiple external accounts connected to same provider", async () => {
+        const activity = {
+            type: "connectExternalAccount",
+            points: 10,
+            providerCategories: ["Device Manufacturer"]
+        } as ConnectExternalAccountActivity;
+        const activityState = {
+            pointsAwarded: 0,
+            providersConnected: []
+        };
+        const newActivityState = await awardConnectExternalAccountActivityPoints(activity, activityState);
+        expect(newActivityState.pointsAwarded).toBe(20);
+        expect(newActivityState.providersConnected).toEqual([11, 22]);
+   });
+
+   it("should append points and update providers despite prior providers no longer being connected", async () => {
+        const activity = {
+            type: "connectExternalAccount",
+            points: 10,
+            providerCategories: ["Health Plan"]
+        } as ConnectExternalAccountActivity;
+        const activityState = {
+            pointsAwarded: 10,
```

Comment:
Minor:  Given the rest of the test setups, I think this would actually be 20 points here to start out if they already had two successful connections to providers 1 and 2.

Reply:
This demonstrates that there is no recalculating of points done based on the providers that had been connected in the past. The description has been updated to reflect the intent of the test

Reply:
Right, I think the description is fine.  What I mean is, just to avoid confusion, the initial state here could be 20 points awarded, since they already have/had providers 1 and 2 connected.  The results below would then be `30` and `[1, 2, 44]`.